libkineto/src/output_json.cpp

```diff
@@ -186,6 +186,9 @@ void ChromeTraceLogger::handleTraceSpan(const TraceSpan& span) {
   if (!traceOf_) {
     return;
   }
+  if (span.startTime == 0 && span.endTime == 0 && span.iteration == -1) {
```

Comment:
confused why this is here? is related to some other change.

Reply:
it's my mistake, removing it